Signal/src/view controllers/PrivacySettingsTableViewController.m

```diff
@@ -106,6 +135,10 @@ - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger
             return 1;
         case PrivacySettingsTableViewControllerSectionIndexCalling:
             return 1;
+        case PrivacySettingsTableViewControllerSectionIndexCallKitEnabled:
+            return self.supportsCallKit ? 1 : 0;
+        case PrivacySettingsTableViewControllerSectionIndexCallKitPrivacy:
+            return (self.supportsCallKit && [[Environment getCurrent].preferences isCallKitEnabled]) ? 1 : 0;
```

Comment:
Done. I wonder if we should move the three calling-related settings to the bottom of the view?